Updates for upgrade to Amazon Linux 2023

maxkadel opened the pull request:
- Use bitnamilegacy image for Redis
- Add more waits to the ssh-keyscan to wait for DNS to resolve in pods

maxkadel committed:
Updates for upgrade to Amazon Linux 2023

- Use bitnamilegacy image for Redis
- Add more waits to the ssh-keyscan to wait for DNS to resolve in pods

maxkadel committed:
Use FQDN for redis

maxkadel committed:
Pin dockerfile to postgres 14, the version on production

It was on "latest", and v18 was breaking.

maxkadel committed:
Change ndots setting to resolve DNS issue

maxkadel committed:
Revert change to redis path

afred reviewed (dismissed):
👍 these look good, although I'm wondering if we still need the alpha,bravo,charlie,delta deployments anymore. These were for migration to widen the pipe into AMS (it appears alpha and bravo were for prod, and charlie and delta were for demo), but I think we are done with that whole process now and we can tighten up the k8s cluster by removing those deployments and dependent resources.

maxkadel committed:
Fix test expectations

maxkadel committed:
Remove unneeded deployments

maxkadel committed:
Fix test

There's a bug in rspec-rails that requires the "include" statement - https://github.com/rspec/rspec-rails/issues/2232

afred reviewed (approved):
👍
